docs/en/roadmap/Introduction.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: global
+title: Alluxio Contribution Roadmap
+group: Roadmap
+priority: 0
+---
+# Alluxio Contribution Roadmap (In Progress)
+
+We warmly welcome you to the Alluxio community. We are excited about your contributions and engagement with our project!
+This guide aims to give you step-by-step instructions on how to get started becoming a contributor to the Alluxio open source project
+and become a qualified independent contributor eventually.
+
+This guide is not a quick start guide but provides a comprehensive roadmap of how to grow from a person beginning with only passion and motivation 
+to a qualified independent contributor that can make useful contributions to open source projects like Alluxio.
+
+If you are a student, considering this to be an internship with industry training.
```

Comment:
considering -> consider and same for below

Reply:
changed, thanks!